### PR TITLE
completed html-forms

### PR DESCRIPTION
**Pull request by Quintin-Russell**
image

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <!--Text form-->
+  <form>
+    <div>
+    <label>Email: <br />
```
Comment (BrettAlbright) [inline html-forms/index.html]: Make sure to add the for attribute here

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    <input type="text" name="email" /></label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: and the id here

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    </div>
+  </form>
+  <!--Dropdown List-->
+  <form>
+    <div>
+      <label>Why is coding cool? <br /></label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: for attribute should be here

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        <select name="reason">
```
Comment (BrettAlbright) [inline html-forms/index.html]: id should be on the select

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small updates needed!

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  <h1>Coding Form!</h1>
+  <!--Text form-->
+  <form>
+    <div>
+    <label for="email">Email: <br />
+    <input id="email" type="text" name="email" /></label>
+    </div>
+  </form>
+  <!--Dropdown List-->
+  <form>
```
Comment (BrettAlbright) [inline html-forms/index.html]: There should be only one `form` element for the entire form

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
update needed.

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**